main.py

```diff
@@ -1,9 +1,17 @@
+import logging
+
+import google.cloud.logging
 from Traidoo.wsgi import application
 
+log_client = google.cloud.logging.Client()
+log_client.setup_logging(log_level=logging.DEBUG)
```

Comment:
Should it be always set to `DEBUG`? Even on production? 🤔

Reply:
I would say until a couple of weeks after go-live.